README.md

```diff
@@ -46,7 +46,7 @@ You can also install the location autocomplete using `npm`:
 
 ```bash
 $ npm install govuk-country-and-territory-autocomplete
-$ ls node_modules/openregister-location-autocomplete/dist/
+$ node_modulesgovuk-country-and-territory-autocomplete/dist/
```

Comment:
Currently missing a slash (it was in the issue as well) and also accidentally loses the `ls` at the start. I think this should be:

```suggestion
$ ls node_modules/govuk-country-and-territory-autocomplete/dist/
```

Reply:
Cheers @36degrees!